resource-validator/src/main/scala/com/broadinstitute/dsp/resourceValidator/ErroredRuntimeChecker.scala

```diff
@@ -30,21 +33,26 @@ object ErroredRuntimeChecker {
         for {
           clusterOpt <- dependencies.dataprocService
             .getCluster(runtime.googleProject, regionName, DataprocClusterName(runtime.runtimeName))
-          _ <- clusterOpt.traverse_ { cluster =>
-            if (cluster.getStatus.getState.name() == "Error")
-              logger.warn(s"${runtime} still exists in Google in Error state. User might want to delete the runtime.")
+          r <- clusterOpt.flatTraverse[F, Runtime] { cluster =>
+            if (cluster.getStatus.getState.name() == "ERROR")
+              logger
+                .warn(s"${runtime} still exists in Google in Error state. User might want to delete the runtime.")
+                .as(none[Runtime])
             else {
               if (isDryRun)
-                logger.warn(
-                  s"${runtime} still exists in ${cluster.getStatus.getState.name()} status. It needs to be deleted."
-                )
+                logger
+                  .warn(
+                    s"${runtime} still exists in ${cluster.getStatus.getState.name()} status. It needs to be deleted."
```

Comment:
This runtime checker is checking runtimes with status other than error as well so should its name reflect that?

Reply:
It only scans runtimes in `Error` status, what do you mean checking runtimes status other than error?

Reply:
Yea, the block starting with [this line](https://github.com/broadinstitute/leonardo-cron-jobs/blob/6bbe169df5d553f3483e1703ca8c0ab6949f3e7c/resource-validator/src/main/scala/com/broadinstitute/dsp/resourceValidator/ErroredRuntimeChecker.scala#L41) seems to check non-errored runtimes.

Reply:
https://github.com/broadinstitute/leonardo-cron-jobs/blob/6bbe169df5d553f3483e1703ca8c0ab6949f3e7c/resource-validator/src/main/scala/com/broadinstitute/dsp/resourceValidator/ErroredRuntimeChecker.scala#L72 is the source of runtimes we scan....the line you're referencing is just action we're taking for these errored runtimes depending on their status in google

Reply:
I realize I was confusing the two. Thanks.